tools/udisksctl.c

```diff
@@ -812,13 +817,25 @@ handle_command_mount_unmount (gint        *argc,
                                               NULL,                       /* GCancellable */
                                               &error))
         {
+          #ifdef HAVE_POLKIT_AGENT_1
```

Comment:
You don't need to ifdef everything that has `polkit` in its name. Instead you should keep the `setup_local_polkit_agent()` and `shutdown_local_polkit_agent()` symbols around and only ifdef their bodies and return dummy value to satisfy its uses. That makes code much cleaner.

In this case you don't need to touch `shutdown_local_polkit_agent()` at all since it tests for `NULL` anyway.